src/components/Player/Profile.svelte

```diff
@@ -59,19 +63,74 @@
 	}
 
 	let editError = null;
-	function onPlayerDataEditError(err) {
-		editError = err?.detail ?? null;
-	}
 
 	let roles = null;
 	function updateRoles(role) {
-		roles = role?.split(',').reverse();
+		roles =
+			role
+				?.split(',')
+				?.reverse()
+				?.filter(r => r?.length) ?? [];
 	}
 
 	function refreshPinnedScores(pinnedScores) {
 		$pinnedScoresStore = pinnedScores ?? [];
 	}
 
+	function onEnableEditModel() {
+		editModel = editModel = {
```

Comment:
Is this some cool trick or just a mistake with two assignments?

Reply:
Just a mistake, probably when ctrl+c/v xD